### DISCUSSION TITLE
Asus ROG G513QY question

### DISCUSSION THREAD

**Discussion by looy17** [seerge/g-helper#2831] [General]
just wondering if ghelper gpu mode is available for the ASUS ROG G15 G513QY as its not showing on mine only "stop gpu application" thanks and sorry if this is dumb 

![Screenshot 2024-07-11 140019](https://github.com/seerge/g-helper/assets/173290694/5c0095e5-3af5-4a2b-ad3a-afb4787d3735)

**Comment by seerge** [comment]
@looy17 if you didn’t have it in AC then you won’t have it here :)
